src/styles/layout/modal.less

```diff
@@ -31,6 +31,18 @@
     width: @modal-width-narrow;
   }
 
+  .icon-button-collection {
+
+    button {
+      height: auto;
+    }
```

Comment:
Sorry for the last minute, but let's target the `.button` class here because we might want to have `a` tags in place of the `button`